packages/ibm-products/src/components/InlineTip/InlineTip.test.js

```diff
@@ -0,0 +1,188 @@
+const linkLabel = `Learn more (${uuidv4()})`;
+const readMoreLabel = `Read more (${uuidv4()})`;
+const readLessLabel = `Read less (${uuidv4()})`;
+const className = `class-${uuidv4()}`;
+const dataTestId = uuidv4();
+
+describe(componentName, () => {
+  it('renders a component InlineTip', () => {
+    render(<InlineTip title={title}>{children}</InlineTip>);
+    expect(screen.getByRole('complementary')).toHaveClass(blockClass);
+  });
+
+  it('has no accessibility violations', async () => {
+    const { container } = render(
+      <InlineTip title={title}>{children}</InlineTip>
+    );
+    await expect(container).toBeAccessible(componentName);
+    await expect(container).toHaveNoAxeViolations();
+  });
+
+  it(`renders children`, () => {
+    render(<InlineTip title={title}>{children}</InlineTip>);
+    screen.getByText(children);
+  });
+
+  it(`renders ghost button`, () => {
+    render(
+      <InlineTip
+        title={title}
+        action={
+          <InlineTipButton onClick={() => {}}>{buttonLabel}</InlineTipButton>
+        }
+      >
+        {children}
+      </InlineTip>
+    );
+    screen.getByText(buttonLabel);
+  });
+
+  it(`renders link`, () => {
+    render(
+      <InlineTip
+        title={title}
+        action={
+          <InlineTipLink href="https://www.ibm.com" target="_blank">
+            {linkLabel}
+          </InlineTipLink>
+        }
+      >
+        {children}
+      </InlineTip>
+    );
+    screen.getByText(linkLabel);
+  });
+
+  it(`renders icon button with 'x' icon`, () => {
+    render(
+      <InlineTip title={title} onClose={() => {}}>
+        {children}
+      </InlineTip>
+    );
+    expect(screen.getByRole('button', { name: /close/i })).toBeInTheDocument();
+  });
+
+  it(`renders tertiary button with 'crossroads' icon`, () => {
+    render(
+      <InlineTip title={title} tertiaryButtonLabel={tertiaryButtonLabel}>
+        {children}
+      </InlineTip>
+    );
+    screen.getByText(tertiaryButtonLabel);
+  });
+
+  it(`renders collapsible, collapsed`, () => {
+    render(
+      <InlineTip
+        title={title}
+        collapsible
+        expandButtonLabel={readMoreLabel}
+        tertiaryButtonLabel={readLessLabel}
+      >
+        {children}
+      </InlineTip>
+    );
+    screen.getByText(readMoreLabel);
+  });
+
+  it(`renders collapsible, expanded`, () => {
+    render(
+      <InlineTip
+        title={title}
+        collapsible
+        expandButtonLabel={readMoreLabel}
+        tertiaryButtonLabel={readLessLabel}
+      >
+        {children}
+      </InlineTip>
+    );
+    userEvent.click(document.querySelector('button'));
+    screen.getByText(readLessLabel);
+  });
+
+  it(`renders an image`, () => {
+    render(
+      <InlineTip
+        title={title}
+        media={{ render: () => <img alt="img" src={InlineTipImage} /> }}
+      >
+        {children}
+      </InlineTip>
+    );
+    expect(document.querySelector('img')).toBeInTheDocument();
```

Comment:
Same comment here regarding `document.querySelector`. If that's the only image you could do something like `screen.getByRole('img');`